268.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * @param {number[]} nums
+ * @return {number}
+ */
+var missingNumber = function (nums) {
+  var sum = 0;
```

Comment:
```suggestion
  let sum = 0;
```
Use `let` instead of `var`